core/block_render_svg_vertical.js

```diff
@@ -354,6 +354,7 @@ Blockly.BlockSvg.prototype.getHeightWidth = function() {
   if (nextBlock) {
```

Comment:
This isn't new, but getHeightWidth is now defined in block_svg.js, block_render_svg_horizontal.js, and block_render_svg_vertical.js.  One of these should be unnecessary.